kore/test/Test/Kore/Step/MockSymbols.hs

```diff
@@ -50,6 +50,7 @@ import qualified Kore.Attribute.Sort.Unit as Attribute
 import qualified Kore.Attribute.Symbol as Attribute
 import qualified Kore.Builtin.Bool as Builtin.Bool
 import qualified Kore.Builtin.Int as Builtin.Int
+import Kore.Builtin.String as Builtin.String
```

Comment:
Why not make this a qualified-only import? Chances are you won't need the `sort -> sort'` change below anymore.